cmds/live.c

```diff
@@ -13,14 +13,22 @@
 #include "utils/socket.h"
 #include "utils/utils.h"
 
-static char *tmp_dirname;
+#define LIVE_NAME "uftrace-live-XXXXXX"
+#define TMP_LIVE_NAME "/tmp/" LIVE_NAME
+
+#define TMP_DIR_NAME_SIZE 32
+
+static char tmp_dirname[TMP_DIR_NAME_SIZE];
 static void cleanup_tempdir(void)
 {
-	if (!tmp_dirname)
+	if (strncmp(TMP_LIVE_NAME, tmp_dirname, strlen(TMP_LIVE_NAME)) != 0 ||
+	    strncmp(LIVE_NAME, tmp_dirname, strlen(LIVE_NAME)) != 0) {
```

Comment:
This condition is always true.  You need to check if it has non-zero contents.

Reply:
Done @namhyung 👍